product_margin_classification/data/decimal_precision.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright (C) 2016-Today GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo noupdate="1">
+
+    <record forcecreate="True" id="decimal_margin" model="decimal.precision">
+        <field name="name">Margin Rate</field>
+        <field name="digits">4</field>
+    </record>
```

Comment:
Why do you create your own decimal precision here in this module? I ask because in another modules of this repo we are still using "Product Price". Does not make sense to use the same precision for all margin fields?